Task execution cleanup throws DB error

With current 2.7.0-SNAPSHOT

Steps:

```
dataflow:>app import --uri https://dataflow.spring.io/task-maven-latest

dataflow:>task create --name b1 --definition timestamp-batch

dataflow:>task launch --name b1 --arguments foo=1
```

Looks like you can't cleanup via shell as UI does request `/tasks/executions/1?action=REMOVE_DATA` and shell:

```
dataflow:>task execution cleanup --id 1

2020-11-11T11:34:46+0000 DEBUG Spring Shell o.a.h.wire:73 - http-outgoing-1 >> "DELETE /tasks/executions/1 HTTP/1.1[\r][\n]"
```

Then server throws error:
```
dataflow-server_1  | 2020-11-11 11:42:56.290 ERROR 1 --- [io-9393-exec-10] o.s.c.d.s.c.RestControllerAdvice         : Caught exception while handling a request
dataflow-server_1  | 
dataflow-server_1  | org.springframework.jdbc.BadSqlGrammarException: PreparedStatementCallback; bad SQL grammar [DELETE BJI FROM BATCH_JOB_INSTANCE BJI WHERE NOT EXISTS ( SELECT JOB_INSTANCE_ID FROM BATCH_JOB_EXECUTION BJE WHERE BJI.JOB_INSTANCE_ID = BJE.JOB_INSTANCE_ID)]; nested exception is org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BJI" not found; SQL statement:
dataflow-server_1  | DELETE BJI FROM BATCH_JOB_INSTANCE BJI WHERE NOT EXISTS ( SELECT JOB_INSTANCE_ID FROM BATCH_JOB_EXECUTION BJE WHERE BJI.JOB_INSTANCE_ID = BJE.JOB_INSTANCE_ID) [42102-200]
dataflow-server_1  | 	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)

```

Just additional note that if you do new launch without changing arguments(usually you get JobInstanceAlreadyCompleteException in a task itself)
```
dataflow:>task launch --name b1 --arguments foo=1
```

Then task errors with npe:
```
Caused by: java.lang.NullPointerException: null
	at org.springframework.batch.core.repository.dao.JdbcExecutionContextDao.getExecutionContext(JdbcExecutionContextDao.java:111) ~[spring-batch-core-4.1.3.RELEASE.jar!/:4.1.3.RELEASE]
	at org.springframework.batch.core.repository.support.SimpleJobRepository.createJobExecution(SimpleJobRepository.java:137) ~[spring-batch-core-4.1.3.RELEASE.jar!/:4.1.3.RELEASE]

```

--- a/spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/JdbcDataflowJobExecutionDao.java
+++ b/spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/JdbcDataflowJobExecutionDao.java
@@ -94,7 +94,7 @@ public class JdbcDataflowJobExecutionDao implements DataflowJobExecutionDao {
 	 * SQL statements for removing Job Instances.
 	 */
 	private static final String  SQL_DELETE_BATCH_JOB_INSTANCE =
-			"DELETE BJI FROM %PREFIX%JOB_INSTANCE BJI " +
+			"DELETE FROM %PREFIX%JOB_INSTANCE BJI " +
 			"WHERE NOT EXISTS ( " +
 			"SELECT JOB_INSTANCE_ID FROM %PREFIX%JOB_EXECUTION BJE WHERE BJI.JOB_INSTANCE_ID = BJE.JOB_INSTANCE_ID)";
 

